packages/bot-web-ui/src/stores/dashboard-store.ts

```diff
@@ -83,16 +84,42 @@ export default class DashboardStore implements IDashboardStore {
             setActiveTour: action.bound,
             setTourDialogVisibility: action.bound,
             setOpenSettings: action.bound,
+            resetTutorialTabContent: action.bound,
+            filterTuotrialTab: action.bound,
             show_toast: observable,
             show_mobile_tour_dialog: observable,
             showVideoDialog: action.bound,
             strategy_save_type: observable,
             toast_message: observable,
+            guide_tab_content: observable,
+            faq_tab_content: observable,
+            video_tab_content: observable,
             setStrategySaveType: action.bound,
             setShowMobileTourDialog: action.bound,
         });
         this.root_store = root_store;
-        this.tutorials_combined_content = [...user_guide_content, ...guide_content, ...faq_content];
+        const removeHTMLTagsFromString = (param = '') => param.replace(/<.*?>/g, '');
+
+        const getUserGuideContent = () => {
+            return [...user_guide_content].map(item => `${item.search_id}# ${item.content.toLowerCase()}`);
+        };
+
+        const getVideoContent = () => {
+            return [...guide_content].map(item => `${item.search_id}# ${item.content.toLowerCase()}`);
+        };
+
+        const getFaqContent = () => {
+            return faq_content.map(item => {
+                return `${item.search_id}# ${item.title.toLowerCase()} ${item.description
+                    .map(inner_item => {
+                        const itemWithoutHTML = removeHTMLTagsFromString(inner_item.content);
+                        return itemWithoutHTML?.toLowerCase();
+                    })
+                    .join(' ')}`;
+            });
+        };
```

Comment:
These three can be variables instead of methods.

Reply:
done ✅